PULL_REQUEST_TEMPLATE

```diff
@@ -3,7 +3,7 @@
 (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
 #### Select relevant component teams: 
 https://github.com/orgs/codice/teams
-#### Choose 2 committers to review/merge the PR. 
+#### Ask 2 committers to review/merge the PR, and tag them here. 
```

Comment:
✏️ Nitpick: I think the comma can be removed.
❓ Where should contributors ask for committers?

Reply:
Good question. I think the ddf-developers Google group would be appropriate.

Reply:
Would the link to the google group be useful to add here?